test/TorchSharpTest/TestTorchTensor.cs

```diff
@@ -8269,5 +8269,111 @@ Tensor nbins_ratio(int seed, int size)
                 }
             }
         }
+
+
+        [Fact(Skip = "Very heavy on the compute")]
```

Comment:
The tests are very heavy on the compute, so I put them in Skip (but I tested them myself)